tensorboard/webapp/widgets/dropdown/dropdown_component.scss

```diff
@@ -20,3 +20,10 @@ mat-select {
   box-sizing: border-box;
   padding: 6px;
 }
+
+mat-select:focus {
```

Comment:
Is this truly good enough? I guess it is irrelevant with current usage but if the dropdown appeared below the `<select>`, you may still want to show the focus ring when you use keyboard to select a `<option>`.

Reply:
imo it is enough.  The default browser `<select>` doesn't show the focus ring when the list of options is opened (via keyboard or mouse).

Reply:
imo it is enough.  The default browser `<select>` doesn't show the focus ring when the list of options is opened (via keyboard or mouse), so we'd be consistent with browser behavior.